### PR TITLE
Created a random agent and moved other agents to a new other_agents directory

### PR DESCRIPTION
**Pull request by tlin010**

### CODE REVIEW

**Comment by quinnmp** [comment]
Do we want to add the new `random_agent` to the README?

File: scripts/other_agents/random_agent.py [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+                    random_positions,  # x, y, z
+                    random_quaternions,  # qw, qx, qy, qz (prototypes are in [w, x, y, z] format)
+                ],
+                dim=1,
+            )
+
+            # Fixed Drop height
+            drop_heights = torch.ones(args_cli.num_envs, device=env.unwrapped.device) * 0.3
+
+            actions[:, 4] = drop_heights
```
Comment (quinnmp) [inline scripts/other_agents/random_agent.py]: Instead of creating a `random_z` tensor of zeros just to ultimately replace it with `drop_heights`, can we just set `random_z = torch.ones(args_cli.num_envs, device=env.unwrapped.device) * 0.3` initially to avoid this? Maybe rename to `drop_heights` as well for clarity

**Review by quinnmp — CHANGES_REQUESTED** [review]

File: scripts/other_agents/random_agent.py (hunk 1 above)
Comment (quinnmp) [reply scripts/other_agents/random_agent.py]: So essentially just replace line 197 with line 212, delete line 214, and then `random_positions = torch.stack([random_x, random_y, drop_heights], dim=1)`.

**Comment by quinnmp** [comment]
Also, given that this is a random agent, we probably want to use the seed:

```
    env_cfg.seed = args_cli.seed
    random.seed(args_cli.seed)
    np.random.seed(args_cli.seed)
    torch.manual_seed(args_cli.seed)
    torch.cuda.manual_seed_all(args_cli.seed)
```

Code from https://github.com/personalrobotics/GCULab/blob/466102bce150d07aedc624b1761f93418df4afea/scripts/bpp_agents/fanwang_bpp_agent.py#L96

**Review by quinnmp — APPROVED** [review]
